todo_rxbloc_app/lib/feature_todos_list/bloc/todos_list_bloc.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rxdart/rxdart.dart';
+import 'package:todos_repository_core/todos_repository_core.dart';
+
+import '../../app_extensions.dart';
+import '../../base/services/todo_service.dart';
+import '../../base/utils/constants.dart';
+
+part 'todos_list_bloc.rxb.g.dart';
+part 'todos_list_bloc_extensions.dart';
+
+abstract class TodosListBlocEvents {
+  void fetchTodosList();
+  void updateTodo(TodoEntity todo);
```

Comment:
As discussed updating and creating todos should not be part of the Todo List Bloc. Think how you can stick better to SRP